server/src/routes/users.ts

```
@@ -70,42 +73,58 @@ router.get('/me', async (req, res, next) => {
+      if (encryptedProfile !== undefined && (!existingUser || encryptedProfile !== existingUser.encryptedProfile)) {
+        // Notify the user themselves (all their active devices)
+        getIo().to(userId).emit('user:updated', { id: updatedUser.id, encryptedProfile: updatedUser.encryptedProfile })
+
+        // Notify contacts
+        const conversations = await prisma.conversation.findMany({
+          where: { participants: { some: { userId } } },
+          include: { participants: { select: { userId: true } } }
+        })
+
+        const recipients = new Set<string>()
+        conversations.forEach(c => c.participants.forEach(p => {
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)